bitnami/mongodb/templates/arbiter/headless-svc.yaml

```diff
@@ -23,8 +23,11 @@ spec:
   clusterIP: None
   ports:
     - name: tcp-mongodb
-      port: {{ .Values.service.port }}
+      port: {{ .Values.arbiter.service.ports.mongodb }}
       targetPort: mongodb
+    {{- if .Values.arbiter.service.extraPorts }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.arbiter.service.extraPorts "context" $) | nindent 4 }}
+    {{- end }}
```

Comment:
This is a headless service, I don't think arbiter nodes needs its own service configuration

Reply:
Good point! I have mixed feelings about this topic. At the moment we are allowing users to add sidecars to the arbiter, so why don't allow then add new ports? If we decide to remove it, I think we should apply the same criteria for the sidecars